### ISSUE TITLE
double keyword argument cause trace

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#591]
In GitLab by @amarao on Jun 12, 2017, 05:59

Installation: sudo apt-get install python-flake8

flake8 --bug-report
```
{
  "dependencies": [
    {
      "dependency": "setuptools",
      "version": "33.1.1"
    }
  ],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.5.3",
    "system": "Linux"
  },
  "plugins": [
    {
      "plugin": "mccabe",
      "version": "0.5.2"
    },
    {
      "plugin": "pycodestyle",
      "version": "2.2.0"
    },
    {
      "plugin": "pyflakes",
      "version": "1.3.0"
    }
  ],
  "version": "3.2.1"
}
```

Bug:

If file contains following line, flake8 crashes.

File:
```
#!/usr/bin/python
foobar(value=1, value=2)
```

Trace:

```
flake8 bad_flake8.py 
Process Process-1:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flake8/checker.py", line 501, in run_ast_checks
    ast = self.processor.build_ast()
  File "/usr/lib/python3/dist-packages/flake8/processor.py", line 211, in build_ast
    return compile(''.join(self.lines), '', 'exec', PyCF_ONLY_AST)
  File "<string>", line 2
SyntaxError: keyword argument repeated

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/multiprocessing/process.py", line 249, in _bootstrap
    self.run()
  File "/usr/lib/python3.5/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3/dist-packages/flake8/checker.py", line 666, in _run_checks_from_queue
    checker.run_checks(results_queue, statistics_queue)
  File "/usr/lib/python3/dist-packages/flake8/checker.py", line 606, in run_checks
    self.run_ast_checks()
  File "/usr/lib/python3/dist-packages/flake8/checker.py", line 504, in run_ast_checks
    row, column = self._extract_syntax_information(exception)
  File "/usr/lib/python3/dist-packages/flake8/checker.py", line 488, in _extract_syntax_information
    lines = physical_line.rstrip('\n').split('\n')
AttributeError: 'NoneType' object has no attribute 'rstrip'
```

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Jun 12, 2017, 06:39

This is a duplicate of https://gitlab.com/pycqa/flake8/issues/279 and was fixed and released in v3.3.0. In the future **please** search closed **and** open issues before opening new ones and try upgrading your version of Flake8.

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Jun 12, 2017, 06:39

closed
